app/src/main/java/org/fossasia/pslab/fragment/LALogicLinesFragment.java

```diff
@@ -66,6 +90,18 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
     public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
         updateLogicLines();
+        YAxis left = logicLinesChart.getAxisLeft();
+        left.setValueFormatter(new ChannelAxisFormatter(channelNames, channelMode));
+        left.setGranularity(1f);
+        left.setTextColor(Color.BLACK);
+        left.setTextSize(12f);
+        logicLinesChart.getAxisRight().setDrawLabels(false);
+
+        /*  For HIDING GRID LINES
```

Comment:
Yes, when I am thinking of adding more settings like hiding grid lines,etc. So that's for it. I